src/main/java/org/jenkinsci/plugins/pubsub/EventProps.java

```diff
@@ -116,6 +135,10 @@
          * Job run status/result.
          */
         job_run_status,
+        /**
+         * Job run SCM commit Id, if relevant.
+         */
+        job_run_commitId,
```

Comment:
Ah thanks for the heads up @i386 and @vivek. I think we were using `BuildData`, but will def go back and revisit that. So @vivek , which one are you saying is the best to use. I'm a little confused by your comment. Sorry 😄 

The actual `commitId` value is not being set by these changes, so I think they're ok. We're just defining the property names here.